adam-cli/src/main/scala/org/bdgenomics/adam/cli/TransformAlignments.scala

```diff
@@ -131,6 +131,8 @@ class TransformAlignmentsArgs extends Args4jBase with ADAMSaveAnyArgs with Parqu
   var storageLevel: String = "MEMORY_ONLY"
   @Args4jOption(required = false, name = "-disable_pg", usage = "Disable writing a new @PG line.")
   var disableProcessingStep = false
+  @Args4jOption(required = false, name = "-save_as_dataset", usage = "EXPERIMENTAL: Use the provided bin size in base pairs to save the data partitioned by genomic range bins using Hive-style partitioning.")
```

Comment:
How about two args, `-partition_by_start_pos` Boolean flag and `-partition_bin_size` Int with a default value?

Reply:
+1 @heuermh